extensions/ingestion/IngestionQueuePopulator.js

```diff
@@ -0,0 +1,73 @@
+const async = require('async');
+const QueuePopulatorExtension =
+          require('../../lib/queuePopulator/QueuePopulatorExtension');
+
+const zookeeper = require('../../lib/clients/zookeeper');
+class IngestionQueuePopulator extends QueuePopulatorExtension {
+    constructor(params) {
+        super(params);
+        this.config = params.config;
+    }
+
+    createZkPath(cb, source) {
+        if (!source || !source.raftCount) {
+            return cb();
+        }
+        const { zookeeperPath } = this.extConfig;
+        this.bucketPrefix = source.prefix ? source.prefix : source.name;
+        return async.times(source.raftCount, (index, next) => {
+            const path = `ingestion/${source.name}/provisions/${index}`;
```

Comment:
path should start with `/` and i think best to use `zookeeperPath` in place of "ingestion" here since the path can be configurable